stubs/python-jose/jose/backends/__init__.pyi

```diff
@@ -0,0 +1,9 @@
+from typing import Any, Callable
```

Comment:
Nit: `Callable` should be imported from `collections.abc`.

Reply:
This has been updated to the correct import